src/main/java/dbProcs/Getter.java

```diff
@@ -209,21 +225,217 @@
 		return result;
 	}
 
+	/**
+	 * This method hashes the user submitted password and sends it to the database.
+	 * The database does the rest of the work, including Brute Force prevention.
+	 * 
+	 * @param userName The submitted user name to be used in authentication process
+	 * @param password The submitted password in plain text to be used in
+	 *                 authentication
+	 * @return A string array made up of nothing or information to be consumed by
+	 *         the initiating authentication process.
+	 */
+
+	public static String[] authUserSSO(String ApplicationRoot, String classId, String userName, String ssoName,
+			String userRole) {
+
+		log.debug("$$$ Getter.authUserSSO $$$");
+
+		log.debug("ssoName = " + ssoName);
+		log.debug("userName = " + userName);
+
+		String[] result = new String[6];
+
+		String userID = new String();
+		String defaultClass = Register.getDefaultClass();
+
+		boolean userFound = false;
+
+		boolean isTempUsername = false;
+
+		Connection conn = Database.getCoreConnection(ApplicationRoot);
+
+		// See if user Exists
+		CallableStatement callstmt;
+		try {
+			callstmt = conn.prepareCall(
+					"SELECT userId, userName, userPass, badLoginCount, tempPassword, classId, suspendedUntil, loginType FROM `users` WHERE ssoName = ? AND loginType='saml'");
+		} catch (SQLException e) {
+			log.fatal("Could create call statement: " + e.toString());
+			throw new RuntimeException(e);
+		}
+
+		log.debug("Gathering userFind ResultSet");
```

Comment:
I think I've fixed this in a later PR. I am considering doing a major setter/getter rewrite to use spring jdbc anyway, that would simplify things quite a bit.

Reply:
Are the changes in this PR included in a later PR? Should this PR be closed? 

It takes a bit of time to review these PR's because of all the formatting churn.

Reply:
Sorry for the extra work with the formatting. 

Yes, my newer PR:s contain this PR. The most recent one is PR #531